### PR TITLE
Update peer

### PR DESCRIPTION
**Pull request by edvinerikson**
cc @taion

### CODE REVIEW

**Comment by taion** [comment]
I don't think you want to do `0.10.x`. I would do either `>=0.9.1` or `^0.9.1 || ^0.10.0`.

**Comment by taion** [comment]
Oh, and bump the dev dependency too.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "relay-subscriptions",
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Comment (taion) [inline package.json:3]: not sure what your publish pipeline is, but doesn't `npm version` usually take care of this for you?

**Review by taion — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (edvinerikson) [reply package.json:3]: yeah I'll run npm publish manually. one less command to run ;)

Comment (taion) [reply package.json:3]: neat, this should fix things for me then. thanks!

**Pull request merged**